src/libraries/Microsoft.Extensions.Configuration/ref/Microsoft.Extensions.Configuration.cs

```diff
@@ -44,7 +44,7 @@ public partial class ConfigurationKeyComparer : System.Collections.Generic.IComp
         public static Microsoft.Extensions.Configuration.ConfigurationKeyComparer Instance { get { throw null; } }
         public int Compare(string? x, string? y) { throw null; }
     }
-    public sealed partial class ConfigurationManager : Microsoft.Extensions.Configuration.IConfiguration, Microsoft.Extensions.Configuration.IConfigurationBuilder, Microsoft.Extensions.Configuration.IConfigurationRoot, System.IDisposable
+    public sealed partial class ConfigurationManager : Microsoft.Extensions.Configuration.IConfiguration, Microsoft.Extensions.Configuration.IConfigurationBuilder, Microsoft.Extensions.Configuration.IConfigurationManager, Microsoft.Extensions.Configuration.IConfigurationRoot, System.IDisposable
```

Comment:
Can't `IConfiguration` and `IConfigurationBuilder` be removed now since it comes transitively from `IConfigurationManager` now?

```suggestion
    public sealed partial class ConfigurationManager : Microsoft.Extensions.Configuration.IConfigurationManager, Microsoft.Extensions.Configuration.IConfigurationRoot, System.IDisposable
```

Reply:
ref .cs files are generated by a tool. This is taking the output of the tool and checking it in. That way the next person who uses the tool doesn't need to fight it.